config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
     get "/", to: "welcome#index"
+    get "/quiz", to: "fear_quiz#new"
+    post "/game-search", to: "game_search#show"
```

Comment:
I wasn't sure what the game search route should be, so I set it to game_search#show. This can easily be changed to route anywhere.

Reply:
I guess I'm still confused what is this game-search? Is this the if I'm user and I search the game and a list of games shows up page? To me that would be an index.

Or is this whatever user searches up, then the game is added to our DB.

Reply:
This is when a non-logged in visitor searches either by keyword or by game from the '/welcome' page. I was thinking if they search by game title it would go to that game's  show page, and if they searched by keyword it would go to an index. I only made it go to #show as a placeholder.